src/components/ThemeToggle/ThemeToggle.stories.tsx

```diff
@@ -29,10 +31,25 @@ export default {
   component: ThemeToggle,
   decorators: [
     (Story, context) => {
-      mockContext.portalUser.theme = context.globals.theme
+      const mocks = [
+        {
+          request: {
+            query: GetUserDocument,
+          },
+          result: () => ({
+            data: {
+              displayName: 'BERNADETTE CAMPBELL',
+              mySpace: [],
+              theme: context.globals.theme,
+            },
+          }),
+        },
+      ]
       return (
         <AuthContext.Provider value={mockContext}>
-          <Story />
+          <MockedProvider mocks={mocks}>
+            <Story />
+          </MockedProvider>
         </AuthContext.Provider>
```

Comment:
These changes were necessary for storybook to show the correct text for the theme toggle button when the theme was changed. This has the side effect of refreshing the entire view, including the storybook manager. It's awkward, but I'm not sure there is another way to properly support it. One option might be to switch to having a dark them and light theme sub story but that would be a big change on how we use storybook currently. Would love to chat about this at a future sync, or here and see what others have to say / think on this.